README.md

```diff
@@ -114,3 +114,45 @@ The interesting fields for this use case are:
   the SSO user ID to be sent to Segment
 - `status.compliantUsername` - Contains the username used in the cluster audit
   log.
+
+## Setting up a containerized Splunk instance
+- Either podman or docker can be user
+- running the following command will deploy a Splunk enterprise container:  
+- `podman pull docker.io/splunk/splunk:latest`
+  `podman run -p 8000:8000 -e "SPLUNK_PASSWORD=AcceptablePassword1" -e "SPLUNK_START_ARGS=--accept-license" -it --name splunk_container docker.io/splunk/splunk:latest`
+- More info about the specific inputs of the command can be found [here|https://docs.splunk.com/Documentation/Splunk/9.0.4/Installation/DeployandrunSplunkEnterpriseinsideDockercontainers]
```

Comment:
Use the [foo][bar] link syntax do make the file readable in text form. Also this syntax is not markdown at all, it looks like the ASCIIDOC syntax used by Jira.

Reply:
leftover, removed